src/test/java/CustomRequestTest.java

```diff
@@ -6,49 +6,28 @@
 
 import org.apache.http.entity.ByteArrayEntity;
 import org.json.JSONObject;
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-import org.mockserver.client.server.MockServerClient;
-import org.mockserver.integration.ClientAndServer;
+import org.junit.jupiter.api.*;
 import org.mockserver.matchers.MatchType;
 import org.mockserver.model.Parameter;
 
-import java.lang.reflect.Field;
 import java.util.Arrays;
 import java.util.EnumSet;
 import java.util.HashMap;
 import java.util.Map;
 
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.mockserver.integration.ClientAndServer.startClientAndServer;
 import static org.mockserver.model.HttpRequest.request;
 import static org.mockserver.model.HttpResponse.response;
 import static org.mockserver.model.JsonBody.json;
 
-public class CustomRequestTest {
-    private static ClientAndServer mockServer;
-
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
The majority of the churn in the test classes is pushing down the repeated mock boilerplate and splitting out a client/server pair for _both_ the API and authentication, since authentication is now handled on a completely different domain.